packages/mgt-components/src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -777,9 +781,10 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
     return html`
        <mgt-flyout light-dismiss class="flyout">
          ${anchor}
-         <div slot="flyout" class="flyout-root" @wheel=${(e: WheelEvent) => this.handleSectionScroll(e)}>
+         <fluent-card slot="flyout" class="flyout-root" @wheel=${(e: WheelEvent) =>
```

Comment:
I tried scrolling my mouse wheel, but it didn't seem to change the selected item, is that what this event is supposed to be doing?

Reply:
The mouse wheel should work when you have a scrollable dropdown as well as a scrollable page. This allows you to scroll in the dropdown independent of the page. Otherwise, the dropdown remains frozen on the document.